From 2ca92aa7dd8589f1d0afc1adda14001bb19744b7 Mon Sep 17 00:00:00 2001
From: Christian Mollekopf <mollekopf@kolabsys.com>
Date: Tue, 14 May 2013 22:20:57 +0000
Subject: Build with cmake 2.8.11

find_package_handle_standard_args already sets SWIG_FOUND, and building
breaks with cmake 2.8.11 otherwise.

BUG: 1723
FIXED-IN: 0.4.3
---
diff --git a/cmake/modules/FindSWIG.cmake b/cmake/modules/FindSWIG.cmake
index f2476a6..010ebb7 100644
--- a/cmake/modules/FindSWIG.cmake
+++ b/cmake/modules/FindSWIG.cmake
@@ -1,9 +1,4 @@
 find_program(SWIG swig /usr/bin/)
-if(SWIG)
-    set(SWIG_FOUND ON)
-    message("SWIG found")
-endif()
 
 #abort if any of the requireds are missing
-find_package_handle_standard_args(SWIG  DEFAULT_MSG
-                                  SWIG_FOUND SWIG)
\ No newline at end of file
+find_package_handle_standard_args(SWIG DEFAULT_MSG SWIG)
--
cgit v0.8.2.1
